templates/prescription/general_edit.html

```diff
@@ -99,204 +86,240 @@
 {/literal}
 </head>
 <body id="prescription_edit" class="body_top">
-
-<form name="prescribe" id="prescribe"  method="post" action="{$FORM_ACTION}" >
-<table>
-    <tr><td class="title"><font><b>{xlt t='Add'}/{xlt t='Edit'}</b></font>&nbsp;</td>
-    <td><a id="save" href=# onclick="submitfun();" class="css_button_small"><span>{xlt t='Save'}</span></a>
-    {if $DRUG_ARRAY_VALUES}
-    &nbsp; &nbsp; &nbsp; &nbsp;
-        {if $prescription->get_refills() >= $prescription->get_dispensation_count()}
-            <input type="submit" name="disp_button"class='css_button_small'  style="margin:0 5px 0 2px;" value="{xla t='Save and Dispense'}" />
-            <input class="input-sm" type="text" name="disp_quantity" size="2" maxlength="10" value="{$DISP_QUANTITY|attr}" />
-            units, {$GBL_CURRENCY_SYMBOL|text}
-            <input class="input-sm" type="text" name="disp_fee" size="5" maxlength="10" value="{$DISP_FEE|attr}" />
-        {else}&nbsp;
-            {xlt t='prescription has reached its limit of'} {$prescription->get_refills()|text} {xlt t='refills'}.
+<div class="container-fluid">
+    <form name="prescribe" id="prescribe" class="form-horizontal" method="post" action="{$FORM_ACTION}">
+        <div class="form-group">
+            <label for="start_date" class="control-label col-sm-2">{xlt t="Starting Date"}</label>
+            <div class="col-sm-2">
+                <input type="text" class="form-control datepicker" name="start_date" id="start_date" value="{$prescription->start_date|oeFormatShortDate|attr}">
+            </div>
+            <div class="col-sm-8">
+                <div class="checkbox">
+                    <label>
+                        <input type="checkbox" name="active" value="1" {if $prescription->get_active() > 0}checked{/if}>
+                        {xlt t="Currently Active"}
+                    </label>
+                </div>
+            </div>
+        </div>
+        <div class="form-group">
+            <label for="drug" class="control-label col-sm-2">{xlt t="Drug"}</label>
+            <div class="col-sm-7">
+                <div class="input-group">
+                    <input type="text" class="form-control" id="drug" name="drug" placeholder="{xlt t='e.x. Aspirin'}" value="{$prescription->drug|attr}">
```

Comment:
sometimes using e.x. and sometimes using ex ?

Reply:
Yeah, I'd prefer `e.x.` but the places I use `ex` are soley for spacing issues - if I used `e.x.` it would push the placeholder text out of the visible area of the textbox